actin/personalization/fetch_tool_from_registry

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+function do_curl() {
+    # New versions (>7.52.1) of `curl` work properly with `--oauth2-bearer` instead of the explicit setting of a header
+    # At time of writing the imager VM was using this older version without prospect of upgrade
```

Comment:
Perhaps this is not relevant for the version you're using? I guess this is working so fine to leave it too, probably limited value to update it.

Reply:
The version on my analysis VM is 7.64, so we may be able to clean this up. What was the context for doing this originally? We could end up using this for other tools, so are we likely to encounter older versions elsewhere?

Reply:
The script is from the imager VM, which was pinned to a much-older version of Debian (version 9 I think) than what the later images use. The reasons for this were just to do with the cost and low priority of updating a VM never exposed outside the organisation.

In fact the original script should also probably be updated as since it was written we have upgraded to Debian 12.